### PR TITLE
docs: make splitter work in IE

### LINKED ISSUE #526: (demo): FlexLayout Splitter demo not working on IE11
FlexBox Splitter not working on IE 11. 

use

#### Usage
```javascript
<div fxLayout="row" ngxSplit="row">
  <div fxFlex="30%" ngxSplitArea>
    Left
  </div>
  <div fxFlex="5px">
    <a href="#" ngxSplitHandle>...</a>
  </div>
  <div fxFlex="70%" ngxSplitArea>
    Right
  </div>
</div>
```

> See https://github.com/angular/flex-layout/issues/266 is not working.

Please provide the steps to resolve the issue.

The main reason for this issue is when we start dragging, after a moment when `SplitAreaDirective.activatedValue` starts returning values which are actually decimal numbers, in IE the value returned is `NaN`.

So, how to resolve the issue?

### PR DESCRIPTION
**Pull request by CaerusKaru**
Fixes #526

### CODE REVIEW

File: src/apps/demo-app/src/app/github-issues/split/split.directive.ts [hunk 1]
```diff
@@ -62,24 +60,17 @@ export class SplitDirective implements AfterContentInit, OnDestroy {
...
-  valueToPixel(value: string | number, parentWidth: number): number {
-    const isPercent = () => String(value).indexOf('px') < 0;
-    let size = parseFloat(String(value));
-    if (isPercent()) {
-      size = parentWidth * (size / 100);  // Convert percentage to actual pixel float value
-    }
-    return size;
-  }
+/** Convert the pixel or percentage value to a raw pixel float value */
+function valueToPixel(value: string, parentWidth: number): number {
```
Comment (ThomasBurleson) [inline src/apps/demo-app/src/app/github-issues/split/split.directive.ts]: This does not work. Consider `valueToPixel('327')` which returns **NaN**.
You need
```ts
function valueToPixel(value: string, parentWidth = 100): number {
   ...
}

Comment (CaerusKaru) [reply src/apps/demo-app/src/app/github-issues/split/split.directive.ts]: `parentWidth` is not optional though?

Comment (ThomasBurleson) [reply src/apps/demo-app/src/app/github-issues/split/split.directive.ts]: True that... I still contend my version is more clear. It is a number and defaults to `100`.

File: src/apps/demo-app/src/app/github-issues/split/split-handle.directive.ts [hunk 2]
```diff
@@ -14,11 +14,24 @@ export class SplitHandleDirective {
   @Output() drag: Observable<{ x: number, y: number }>;
 
   constructor(ref: ElementRef, @Inject(DOCUMENT) _document: any) {
-    const getMouseEventPosition = (event: MouseEvent) => ({x: event.movementX, y: event.movementY});
+    let prevX = 0;
```
Comment (ThomasBurleson) [inline src/apps/demo-app/src/app/github-issues/split/split-handle.directive.ts]: I suggest this is a improved version:

```ts
  let last = {x: 0, y: 0};
  const getPositionDelta = (event: MouseEvent) => { 
    const current = {x: event.screenX, y: event.screenY};   
    const delta = { x: current.x - last.x, y: current.y - last.y };
    
    last = current;
    return delta;
  }
  
  const mousedown$ = fromEvent(target, 'mousedown').pipe( map(getPositionDelta) );
  const mousemove$ = fromEvent(window, 'mousemove').pipe( throttleTime(50), map(getPositionDelta) );
  const mouseup$ = fromEvent(window, 'mouseup');

  return mousedown$.pipe(switchMap(() => mousemove$.pipe( takeUntil(mouseup$) )));
```

**Pull request merged**